pkgs/development/python-modules/tensorflow/default.nix

```diff
@@ -529,29 +567,26 @@ in buildPythonPackage {
   # TEST_PACKAGES in tensorflow/tools/pip_package/setup.py
   nativeCheckInputs = [
     dill
-    keras
     portpicker
     tblib
   ];
+  # TODO write simple test without keras, or reactivate upstream tests
```

Comment:
It looks like this is done now? AFAICT the test case below doesn't include any keras.

Reply:
I had left it there as a reminder to move the old test into `keras`. But the `keras` package moved to KERAS 3 in the meantime, which is fully multi-backend. So this is probably unneeded by now.